ocrd/ocrd/workspace.py

```diff
@@ -851,7 +852,15 @@ def save_image_file(self, image,
         if not force and self.overwrite_mode:
             force = True
         image_bytes = io.BytesIO()
-        image.save(image_bytes, format=MIME_TO_PIL[mimetype])
+        if dpi and not isinstance(dpi, tuple):
+            dpi = (dpi, dpi)
+        elif not dpi:
+            # TODO - brittle, will this always find the original image?
+            orig_img_file = self.mets.find_files(pageId=page_id, mimetype='//^image')[0]
+            orig_img_pil = self.resolve_image_exif(orig_img_file.url)
+            exif = OcrdExif(orig_img_pil)
+            dpi = (exif.xResolution, exif.yResolution)
```

Comment:
Should probably also check `exif.resolutionUnit == 'inches'` or divide by 2.54.

Reply:
552a7c9